crates/emitter/src/atomset.rs

```diff
@@ -0,0 +1,55 @@
+use std::collections::HashMap;
```

Comment:
Shouldn't this file be removed?

Reply:
why?

Reply:
Forget it, I thought this was replaced by `AtomList`

Reply:
yeah, the naming here is really bad.
if you have idea for better naming, let me know.